src/main/scala/com/lightbend/rp/sbtreactiveapp/SbtReactiveApp.scala

```diff
@@ -161,6 +163,6 @@ object SbtReactiveApp {
         .toSeq
         .map(ns("sbt-reactive-app-version") -> _)
 
-    keyValuePairs.toMap
+    keyValuePairs.toVector
```

Comment:
I'd either stick to `keyValuePairs` or if need dedupe + order `ListMap(keyValuePairs : _*)`

Reply:
I think this is fine. The alternative would be to replace `.toSeq` with `.toVector` in this statement, thus omitting the need for the call here on L166 (as `.toSeq` is `scala.collection.Seq`, not `scala.collection.immutable.Seq`)